cmd/completion/main.go

```diff
@@ -3,20 +3,28 @@ package main
 import (
 	"context"
 	"flag"
+	"fmt"
```

Comment:
There is alot in there these days. Might make sense to explore moving it out of main so it can be better tested.

Reply:
a lot of formatting?

Reply:
Oh. I just picked the fist line. I meant the whole file itself.

Reply:
attempted to improve this